packages/context-menu/src/vaadin-context-menu-overlay.js

```diff
@@ -101,13 +102,32 @@ class ContextMenuOverlay extends OverlayElement {
     return {
       xMax: overlayRect.right - contentRect.width,
       xMin: overlayRect.left + contentRect.width,
-      yMax,
-      left: overlayRect.left,
-      right: overlayRect.right,
-      top: overlayRect.top,
-      width: contentRect.width
+      yMax
     };
   }
+
+  _updatePosition() {
+    this.style.transform = '';
+    super._updatePosition();
+
+    const isFullScreen = this.offsetWidth === window.innerWidth;
```

Comment:
Could we use the logic provided by `vaadin-device-detector` instead? This check looks a bit unreliable: it relies on CSS, but the corresponding styles for `[phone]` are only set by themes, not `src`.

Reply:
Fullscreen detection no longer necessary due to https://github.com/vaadin/web-components/pull/2917#discussion_r743449412